src/codec_aom.c

```diff
@@ -532,10 +532,7 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
                                       avifCodecEncodeOutput * output)
 {
     struct aom_codec_enc_cfg * cfg = &codec->internal->cfg;
-    aom_codec_iface_t * encoderInterface = NULL;
-    unsigned int aomUsage = AOM_USAGE_GOOD_QUALITY;
-    int aomCpuUsed = -1;
-    avifBool lossless = AVIF_FALSE;
+    avifBool quantizerUpdated = AVIF_FALSE;
```

Comment:
I restored this function to its original structure. There is only one `if (!codec->internal->encoderInitialized)` block now (at line 537).